### DISCUSSION TITLE
How to install a FIPS compliant Velero version in redhat clusters?

### DISCUSSION THREAD

**Discussion by Naveen-Kamagani** [vmware-tanzu/velero#7449] [Community support Q&A]
How to install a FIPS compliant Velero version in redhat clusters?
Is there any documentation for the above process ?

**Comment by kaovilai** [comment]
You can install [OADP Operator](https://catalog.redhat.com/software/container-stacks/detail/646f70952ca122cdf3314a83) which is available in the OperatorHub Marketplace.

This operator ships FIPS complaint velero per v1.3 [release notes](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/backup_and_restore/index).
